tracks/julia/exercises/pythagorean-triplet/mentoring.md

```diff
@@ -5,14 +5,44 @@
 
 ## Example solutions
 
-A O(n^2) time algorithm:
-
 ````
+There are three reasonably common solutions to this problem, a cubic time solution, a quadratic time solution and a linear time solution.
```

Comment:
Could you perhaps add a TL;DR of what "cubic time" etc mean? The terms may not be familiar to people without a CS/num. maths background.

Reply:
(This could be a follow-up PR, or some general mentoring note unspecific to this exercise)

Reply:
Do you think O(n^3), O(n^2), etc is clearer? Could just use those.

Reply:
I think the same applies to those notations. Not fully sure though